historical-social-research.csl

```diff
@@ -0,0 +1,628 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="display-and-sort" page-range-format="chicago">
+  <!-- Polyglot; journal publishes in English and German -->
+  <info>
+    <title>Historical Social Research</title>
+    <title-short>HSR</title-short>
+    <id>http://www.zotero.org/styles/historical-social-research</id>
+    <link href="http://www.zotero.org/styles/historical-social-research" rel="self"/>
+    <link href="http://www.zotero.org/styles/historical-social-research" rel="template"/>
+    <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
```

Comment:
@POBrien333, do you remember which Chicago variant you used here? The "template" link isn't right. The "documentation" link should also be updated (probably to https://www.gesis.org/en/hsr/faqs/faqs-for-authors/guidelines/).